native/packages/react-native-bpk-theming/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import BpkThemeProvider, { bpkWithTheme } from './src/BpkThemeProvider';
+
+ export default BpkThemeProvider;
+ export { bpkWithTheme };
```

Comment:
I reckon `withTheme` is fine ?

Reply:
Ok sure - I wasn't sure what our policy was on duplicating existing names. I suppose with named imports there's less chance of collisions though.